xml/hwbp_registers.xml

```diff
@@ -967,6 +967,14 @@
         tie any number of upper bits to 0. The $|select|$ bits may only support
         0 (ignore).
 
+        Byte-granular comparison of the scontext CSR to the trigger's svalue in
+        textra32 and textra64 registers allows scontext to be defined to include
+        more than one element of comparison.  For example, software instrumentation
+        can program the scontext value to be the concatenation of different ID contexts
+        such as process ID and thread ID.  The user can then program byte compares of
+        the trigger to include one or more of the contexts in the compare. For textra32
+        there are two byte-compare mask bits; for textra64 there are 5 byte-compare mask bits.
```

Comment:
What about when sselect==2 (ASID comparison)?  Would the byte mask apply in that case?  This text only describes scontext comparison and needs to explicitly state whether the masking applies to ASID.